examples/asr/transcribe_speech.py

```diff
@@ -242,17 +257,46 @@ def autocast():
     if type(transcriptions) == tuple and len(transcriptions) == 2:
         transcriptions = transcriptions[0]
     # write audio transcriptions
+
+    if cfg.append_pred:
+        logging.info(f'Transcripts will be written in "{cfg.output_filename}" file')
+        if cfg.pred_name_postfix is not None:
+            pred_by_model_name = cfg.pred_name_postfix
+        else:
+            pred_by_model_name = model_name
+        pred_text_attr_name = 'pred_text_' + pred_by_model_name
+    else:
+        pred_text_attr_name = 'pred_text'
     with open(cfg.output_filename, 'w', encoding='utf-8') as f:
         if cfg.audio_dir is not None:
             for idx, text in enumerate(transcriptions):
-                item = {'audio_filepath': filepaths[idx], 'pred_text': text}
+                item = {'audio_filepath': filepaths[idx], pred_text_attr_name: text}
                 f.write(json.dumps(item) + "\n")
         else:
             with open(cfg.dataset_manifest, 'r') as fr:
                 for idx, line in enumerate(fr):
                     item = json.loads(line)
-                    item['pred_text'] = transcriptions[idx]
+                    item[pred_text_attr_name] = transcriptions[idx]
                     f.write(json.dumps(item) + "\n")
+        # with open(cfg.output_filename, 'r', encoding='utf-8') as fo:
```

Comment:
Please remove the commented-out code from the final version.

Reply:
+